File: backend/database/models/trailingstop.js
'use strict';
module.exports = (sequelize, DataTypes) => {
  const trailingStop = sequelize.define('trailingStop', {
    percent: {
      type: DataTypes.DECIMAL,
      allowNull: false,
      validate: {
        max: {
          args: 1,
          msg: 'Value of trailing stop may not be greater than 1.'
        },
        notNull: {
          msg: 'Value can not be null.'
        },
        isDecimal: {
          args: true,
          msg: 'Trailing stop must be a decimal.',
        },
      }
    }
  }, {
    freezeTableName: true,
    tableName: 'trailingStop',
  });
  trailingStop.associate = function(models) {
    trailingStop.belongsTo(models.position, {foreignKey: 'positionId'});
  };
  return trailingStop;
};

File: backendSharp/Models/TrailingStop.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;


namespace Trowoo.Models
{
    public class TrailingStop
    {
        public int Id {get; set;}

        [Required]
        [Range(0,1)]
        public decimal Percent {get; set;}

        [Required]
        [ForeignKey("PositionId")]
        public int PositionId {get; set;}

    }
}